embedded-artifactory/src/test/resources/bootstrap.yaml

```diff
@@ -0,0 +1,3 @@
+embedded:
```

Comment:
and what about end users? if artifactory is slow on startup -- use this as default config in the library, not only for tests

Reply:
Moved config to use as default config